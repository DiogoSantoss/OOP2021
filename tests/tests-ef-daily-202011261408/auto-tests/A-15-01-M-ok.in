9
1
50
0
0
