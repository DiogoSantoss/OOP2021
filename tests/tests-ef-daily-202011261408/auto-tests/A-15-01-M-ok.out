Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Preço: Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 